frontend/src/Frontend.hs

```diff
@@ -113,6 +113,7 @@ networkDispatch route ndbs netId = prerender_ blank $ do
               f = maximum . map _tipHeight . HM.elems . _cutChains
           height <- f <$$$> getCut (ch <$ pb)
           void $ networkHold (inlineLoader "Getting latest cut...") (mainPageWidget netId <$> height)
+        NetRoute_Search -> searchPageWidget netId
```

Comment:
We're introducing this new page but I don't see any way of getting to that page from any of the existing pages.  What do you have in mind for that?

Reply:
As discussed we're leaving it nav-less as it's a WIP